source/AI.cpp

```diff
@@ -2949,7 +2949,7 @@ void AI::AutoFire(const Ship &ship, Command &command, bool secondary) const
 		{
 			// NPCs shoot ships that they just plundered.
 			bool hasBoarded = !ship.IsYours() && Has(ship, currentTarget, ShipEvent::BOARD);
-			if(currentTarget->IsDisabled() && spareDisabled && !hasBoarded && !disabledOverride)
+			if(currentTarget->IsDisabled() && spareDisabled && (!hasBoarded || person.Disables()) && !disabledOverride)
```

Comment:
`spareDisabled = person.Disables() || (person.Plunders() && ship.Cargo().Free())`, so we basically have `person.Disables()` in here twice. 

Reducing the booleans to letters, we have `A && (B || (C && D)) && (!E || B)`. (Ignoring disabledOverride since it's a player command thing.) That has the same truth table as `A && (B || (C && D && !E)`.

So maybe replace the `spareDisabled` variable with a `plunders` and `disables` variable, where `bool plunders = person.Plunders() && ship.Cargo().Free();` and `bool disables = person.Disables();`. Then this would become the following here and below:
```suggestion
			if(currentTarget->IsDisabled() && (disables || (plunders && !hasBoarded) && !disabledOverride)
```